test/unit/AssetManagedPair.t.sol

```diff
@@ -208,8 +208,8 @@ contract AssetManagedPairTest is BaseTest {
 
     function testMint_AfterLoss(uint256 aNewManagedBalance0, uint256 aNewManagedBalance1) external allPairs {
         // assume
-        uint256 lNewManagedBalance0 = bound(aNewManagedBalance0, 1, 10e18);
-        uint256 lNewManagedBalance1 = bound(aNewManagedBalance1, 1, 10e18);
+        uint256 lNewManagedBalance0 = bound(aNewManagedBalance0, 1, 9e18);
+        uint256 lNewManagedBalance1 = bound(aNewManagedBalance1, 1, 9e18);
 
```

Comment:
What was the reason fro 10 -> 9?

Reply:
Reason for this is: due to the update for foundry the fuzzes are getting more coverage at the extreme values. So sometimes if it gets to `10e18` then there is no loss for the pair, and the redeemer does not get back fewer tokens, making the assert condition below fail. 

I lowered the range to `9e18` to make sure there's a loss for the pair, in line with the spirit of the test. Else the alternative would be to change the assert condition from `assertLt` to `assertLte`, which would be smellier code.

Reply:
Makes sense